### PR TITLE
[NOREF] Migration Generation Utility: scripts/dev genmigration

### PR DESCRIPTION
**Pull request by OddTomBrooks**
Added genmigration to scripts/dev which generates a migration prefixed with the current UTC timestamp in YYYYmmddHHMMSS format.

# NOREF

## Changes and Description

- Added scripts/dev genmigration

## How to test this change

- Call scripts/dev genmigration
- Observe that a migration is generated with the format "v%Y%m%d%H%M%S__.sql"

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: scripts/dev [hunk 1]
```diff
@@ -471,6 +472,12 @@ task :lint do
   sh "pre-commit run -a"
 end
 
+desc "Generate a new migration with the current utc timestamp prefix"
+task genmigration: :consume_args do |t, args|
+  migrationName = Time.now.utc.strftime("v%Y%m%d%H%M%S__.sql")
```
Comment (ClayBenson94) [inline scripts/dev:477]: Not sure if flyway actually cares or not, but for consistency can you make this a capital V?

```suggestion
  migrationName = Time.now.utc.strftime("V%Y%m%d%H%M%S__.sql")
```

**Review by ClayBenson94 — CHANGES_REQUESTED** [review]

File: scripts/dev (hunk 1 above)
Comment (OddTomBrooks) [reply scripts/dev]: Done

**Comment by StevenWadeOddball** [comment]
Thanks for making a proof of concept for this Tom. From conversation outside of this channel, I think we are leaning towards closing this PR in favor of addressing the underlying issue of migration conflicts. I think it's ok to close this out.

**Review by StevenWadeOddball — CHANGES_REQUESTED** [review]
Hey Tom,

Thanks again for trying this out. The code here works fine and looks good, however, I don't think we should go with this approach. I think we should lean on GitHub configuration like Clay suggested (Which would mean we don't use this utility). Mind closing this out?

**Pull request closed without merging**